src/api/app/controllers/webui/user_controller.rb

```diff
@@ -22,7 +22,8 @@ def show
     @role_titles = @displayed_user.roles.global.pluck(:title)
     @account_edit_link = CONFIG['proxy_auth_account_page']
 
-    return unless switch_to_webui2
+    return unless switch_to_webui2 && CONFIG['contribution_graph'].present?
```

Comment:
Hm I think for the rest of the switches we use `:on`|`:off` as values.

Reply:
I have also change the docu in the wiki now...

Reply:
@hennevogel done using `:on | :off`. I also fixed one failing test.